recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-h3ulcb-dom0.dts

```diff
@@ -52,11 +52,10 @@
 	 * CPU nodes to be present get broken. This is why we have to explicitly
 	 * remove the following.
 	 */
-	soc {
-		/delete-node/thermal-zones;
-		/delete-node/pmu_a57;
-		/delete-node/pmu_a53;
-	};
+	/delete-node/thermal-zones;
```

Comment:
For the whole patch: it does need description and rationale for what is done

Reply:
ok, will do

Reply:
ok, will provide